modules/030-cloud-provider-azure/openapi/values.yaml

```diff
@@ -0,0 +1,79 @@
+x-extend:
+  schema: config-values.yaml
+type: object
```

Comment:
I think better way use links to cloud-provides specs for exclude copy-paste like [yandex](https://github.com/deckhouse/deckhouse/blob/main/modules/030-cloud-provider-yandex/openapi/values.yaml#L19).

Reply:
https://github.com/deckhouse/deckhouse/blob/main/candi/cloud-providers/yandex/openapi/cluster_configuration.yaml#L7